src/components/tl/ucp/tl_ucp_context.c

```diff
@@ -137,6 +137,34 @@ UCC_CLASS_CLEANUP_FUNC(ucc_tl_ucp_context_t)
 
 UCC_CLASS_DEFINE(ucc_tl_ucp_context_t, ucc_tl_context_t);
 
+ucc_status_t ucc_tl_ucp_mem_map(void *addr, size_t length, ucs_memory_type_t mem_type,
+                                ucc_tl_ucp_context_t *ctx)
+{
+    ucp_mem_map_params_t mmap_params;
+    ucp_mem_h mh;
```

Comment:
vars are not aligned (you can run clang on the commit diff with: git-clang-format-11  HEAD~1 --style file)

Reply:
fixed